Formula/jack.rb

```diff
@@ -86,6 +66,23 @@ def plist
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jackd --version")
+    source_name = "test_source"
+    sink_name = "test_sink"
+    fork do
+      system "#{bin}/jackd", "-X", "coremidi", "-d", "dummy"
```

Comment:
I believe these should be 
```suggestion
      exec "#{bin}/jackd", "-X", "coremidi", "-d", "dummy"
```